lib/connection/connection.js

```diff
@@ -127,6 +139,11 @@ function Connection(context)
       !Util.exists(callback) || Util.isFunction(callback),
       ErrorCodes.ERR_CONN_CONNECT_INVALID_CALLBACK);
 
+    if (connectionConfig.host.endsWith(".privatelink.snowflakecomputing.com"))
```

Comment:
nit: make this into a constant and use the constant here and below

Reply:
Ok, constant for privatelink suffix is now made